### PR TITLE
RD-6533 Adjusted buttons appearance

### PR DESCRIPTION
**Pull request by Vorbert-Kruk**
## Description

Changes consists of modifying button colors and variations, so that the application presentation would be more consistent across its views 🎉 

### Before
Create environment button
Create service button
Blueprint catalog
Deploy modal

### After
Create environment button
Create service button
Blueprint catalog
Deploy modal

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [ ] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

I've went over test files related to the modified places, to see if adjustment would be needed
Fortunately it doesn't appear so! 🎉 
But just to be cautious, I've executed system tests for related test scenarios 🚀 

1. Execution
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4288/)
`deployment_button_spec` failed due to a random timeout - it was experienced while working on previous PRs and it's not related to the changes introduced within this PR - I'll notify UI Guardian about it

2. Execution - just to make sure that every test is passing 🎉 
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4293/)
Same situation as with the above text execution

3. Execution - just to make sure that every test is passing 🎉 
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4294/)
Great success 🎉 

## Documentation

Will take place as a part of RD-6607

### CODE REVIEW

File: widgets/deploymentButton/src/widget.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ Stage.defineWidget({
             id: 'color',
             name: tConfiguration('color.name'),
             description: tConfiguration('color.description'),
-            default: 'green',
+            default: 'blue',
```
Comment (qooban) [inline widgets/deploymentButton/src/widget.tsx:40]: I know you'll be documenting all changes coming out from theming as part of RD-6607. 

Depending on your preferences, maybe worth to add a note there not to forget about this specific thing. As I think there'll be a lot screenshots to update and we may miss such small details.

**Review by qooban — APPROVED** [review]
Looks good 👍🏻 

I saw you executed system tests only for certain test suites. If you fell comfortable with it, then all good. I was just wondering wouldn't it be worth to run full system test to be fully sure we are not braking anything. Leaving it to you.

File: widgets/deploymentButton/src/widget.tsx (hunk 1 above)
Comment (Vorbert-Kruk) [reply widgets/deploymentButton/src/widget.tsx:40]: Sure, seems reasonable :) 
Comment was added to the RD-6607 🎉

**Comment by Vorbert-Kruk** [comment]
@qooban as system tests has passed successfully, I'll merge this PR 🎉

**Pull request merged**